packages/builders/src/interactions/slashCommands/mixins/NameAndDescription.ts

```diff
@@ -31,4 +35,76 @@ export class SharedNameAndDescription {
 
 		return this;
 	}
+
+	/**
+	 * Sets a name localization
+	 *
+	 * @param locale The locale to set a description for
+	 * @param localizedName The localized description for the given locale
+	 */
+	public setNameLocalization(locale: LocaleString, localizedName: string) {
```

Comment:
Should this support setting to null/undefined?

```suggestion
	public setNameLocalization(locale: LocaleString, localizedName: string | null) {
```